CONTRIBUTING.md

```diff
@@ -13,9 +13,18 @@ All contributors must agree to the Contributor Agreement ([ContributorAgreement.
 
 General guidelines:
 
-  * The signac development is based on the [git flow model][gitflow], which means new features should be developed within a feature branch based off the 'develop' branch.
-  * If external library depedencies cannot be avoided, they must be added as *soft depedencies*.
-  * All contributed code should pass `flake8` checks as specified in the tox.ini configuration file.
-  * All new features require unit tests.
+  * Use the [git flow][git flow] model of development:
```

Comment:
Not sure if this link is going to work - the reference lacks a space, but this line has a space.

Reply:
I've just replaced all links with the standard inline formatting.